src/Microsoft.SqlTools.ServiceLayer/TableDesigner/TableDesignerService.cs

```diff
@@ -816,6 +828,14 @@ private TableViewModel GetTableViewModel(TableInfo tableInfo)
                 columnViewModel.IdentityIncrement.Enabled = column.CanEditIdentityValues;
                 columnViewModel.IdentityIncrement.Value = column.IdentityIncrement?.ToString();
                 columnViewModel.CanBeDeleted = column.CanBeDeleted;
+                columnViewModel.IsComputed.Enabled = column.CanEditIsComputed;
+                columnViewModel.IsComputed.Checked = column.IsComputed;
+                columnViewModel.ComputedFormula.Enabled = column.CanEditComputedFormula;
+                columnViewModel.ComputedFormula.Value = column.ComputedFormula;
+                columnViewModel.IsComputedPersisted.Enabled = column.CanEditIsComputedPersisted;
+                columnViewModel.IsComputedPersisted.Checked = column.IsComputedPersisted == true;
```

Comment:
you don't need the `==true` part.

Reply:
I defined `IsComputedPersisted ` in DacFx as `bool?` and it will be `null` if the column is not computed.